packages/manager/modules/common-translations/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "lib": ["dom", "es2021"],
+    "target": "es2021",
+    "types": ["vite/client", "node"],
+    "module": "ES2020",
+    "outDir": "dist",
+    "allowJs": false,
+    "skipLibCheck": true,
+    "esModuleInterop": false,
+    "allowSyntheticDefaultImports": true,
+    "moduleResolution": "Node",
+    "isolatedModules": true,
+    "declaration": true,
+    "declarationDir": "./dist/types",
```

Comment:
Are we emitting some `*.d.ts` files in this library?

Reply:
Yes.